podcasts/NotificationsViewController.swift

```diff
@@ -30,45 +71,73 @@ class NotificationsViewController: PCViewController, UITableViewDataSource, UITa
         settingsTable.reloadData()
     }
 
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return FeatureFlag.notificationsRevamp.enabled ? 3 : 1
+    }
+
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        NotificationsHelper.shared.pushEnabled() ? 3 : 1
+        guard let sectionType = Section(rawValue: section) else {
+            return 0
+        }
+        switch sectionType {
+        case .episodes:
+            return NotificationsHelper.shared.pushEnabled() ? 3 : 1
+        case .featuresAndOffers:
+            return rows[section].count
+        case .recommendationsAndReminders:
```

Comment:
Could these 2 be on the same case? `case .featuresAndOffers, .recommendationsAndReminders:`

Reply:
Addressed here: https://github.com/Automattic/pocket-casts-ios/pull/2934/commits/864279377f804ac8498a4eb916856a2e5934ed79